README.md

```diff
@@ -133,3 +133,55 @@ var intValue: Int
 var intValue : Int
 
 ```
+
+## Best Practices ##
+
+### Property Observers ###
+Cf Apple documentation:
+
+```
+Property observers observe and respond to changes in a property’s value. Property observers are called every time a property’s value is set, even if the new value is the same as the property’s current value.
+```
+
+You can use 2 types of property observers:
+
+* *willSet* is called just before the property value is stored.
+* *didSet* is called right after the property value is stored.
+
+```swift
+var intValue: Int = 0 {
+	willSet(newIntValue) {
```

Comment:
Don't need to use `newIntValue`. It should just be `willSet ` and use `newValue`